2 - RTB/Documentazione interna/Analisi dei rischi/Analisi dei rischi.typ

```diff
@@ -0,0 +1,138 @@
+#import "/template.typ": *
+
+#show: project.with(
+  title: "Analisi dei rischi",
+  subTitle: "",
+  authors: (
+    "Banzato Alessio",
+    "Nardo Silvio",
+    "Oseliero Antonio",
+  ),
+  showLog: true,
+);
+
+= Profili di rischio
+
+A ciascun rischio individuato si associano:
+  - informazioni descrittive sul suo contesto;
+  - impatto: può essere lieve, medio o grave. Esprime l'ordine di grandezza dell'effetto generato dall'evento;
+  - probabilità: da 1 a 5. Esprime la probabilità di verificarsi del rischio;
+  - soglie di accettazione del rischio;
+  - azioni previste in funzione delle soglie, possono includere:
+    - l'eliminazione del rischio;
+    - la riduzione della sua probabilità o gravità;
+    - l'accettazione del rischio.
+
+  - conseguenze relative a ciuascuna delle azioni possibili, esse descrivono gli effetti collaterali a breve o medio termine che il rischio può comportare.
+
+= Rischi
+
+== Comunicazione con il Proponente
+I contatti con il Proponente subiscono variazioni nella qualità e nella frequenza a causa di problematiche fuori dal controllo del gruppo. Questa situazione potrebbe causare un rallentamento significativo del lavoro, soprattutto durante l'analisi dei requisiti.
+
+- Impatto: grave;
+- Probabilità: 1;
+- Soglie: 
+  - se il periodo critico previsto è sufficientemente breve da permettere al gruppo di continuare con un corretto avanzamento dei lavori nonostante la variazione nella comunicazione con il Proponente, si adotta l'"Accettazione del rischio" mantenendo monitorata la situazione;
+  - se il periodo critico previsto può mettere a rischio il corretto avanzamento dei lavori o la qualità degli stessi, si adotta l'azione correttiva di rischio: "Riduzione della sua probabilità o gravità".
+
+=== Opzioni di trattamento del rischio
+==== Riduzione della sua probabilità o gravità
+*Strategia di trattamento* \
+Si possono adottare, in accordo con il Proponente, una o più tra le seguenti strategie comunicative:
+  - uso di strumenti asincroni per facilitare lo scambio di informazioni tra gruppo e Proponente;
+  - pianificazione anticipata degli incontri di revisione dell'avanzamento;
+  - programmazione di incontri periodici di aggiornamento, anche brevi.
+\ *Conseguenze* \
+È prevista una modifica sostanziale, più o meno temporanea, nella comunicazione con il Proponente. Questo potrebbe portare a ritardi nei lavori dovuti ad un maggior impegno da parte del gruppo nel garantire l'adozione delle nuove pratiche.
+
+==== Accettazione del rischio
+*Strategia di trattamento* \
+Si attende il termine del periodo nel quale la comunicazione con il Proponente risulta problematica.
+Per evitare perdite di tempo il gruppo continua a lavorare priorizzando task che non dipendono direttamente dall'intervento del Proponente.
+In caso di accettazione del rischio è importante tenere monitorata nel tempo la situazione per accertarsi che le circostanze non richiedano una modifica nell'approccio di risoluzione.
```

Comment:
"accettazione"  da cambiare in "Accettazione", nel resto del documento è scritto sempre con la "a" maiuscola